projects/wallet-template/src/containers/WalletPopup/pages/Accounts/AddAccount/index.tsx

```diff
@@ -22,11 +22,11 @@ type FormFields = {
 export const AddAccount = () => {
   const navigate = useNavigate()
   const [mnemonic, _] = useState(generateMnemonic(256).split(" "))
-  const { data: keysets, isLoading: isKeysetsLoading } = useSWR(
+  const { mutate } = useSWRConfig()
+  const { data: keysets, isLoading: areKeysetsLoading } = useSWR(
```

Comment:
same comment as before, use `useSWR` bounded `mutate` instead of the `useSWRConfig` `mutate`